src/index.html

```diff
@@ -11,7 +11,190 @@
+            type="date"
+            name="date" 
+            required
+          >
+        </label>
+
+        <label for="checkbox" class="input-label">
+          I accept the term of the agreement
+          <input 
+            id="checkbox" 
+            type="checkbox" 
+            name="agreement" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="regist fieldset">
+        <legend>Registration</legend>
+
+        <label for="email" class="input-label">
+          E-mail:
+          <input 
+            id="email" 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            required 
+            multiple
+          >
+        </label>
+
+        <label for="password" class="input-label">
+          Password:
+          <input 
+            id="password" 
+            type="password" 
+            name="password" 
+            minlength="10" 
+            maxlength="20" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fact fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label for="radio" class="input-label">
+          Do you love cats?
+          <input 
+            id="radio"
+            type="radio"
+            name="radio"
+            value="y"
+          > 
+            Yes
+          <input 
+            id="radio1" 
+            type="radio" 
+            name="radio" 
+            value="n"
+          >
+            No
+        </label>
+        </div>
+
+        <label for="color" class="input-label">
+          What is your favorite color?
+          <input 
+            id="color" 
+            type="color"
+            name="color" 
+            required
+          >
+        </label>
+        
+        <label for="time" class="input-label">
+          What time do you go to bed?
+          <input 
+            id="time"
+            type="time" 
+            name="time" 
+            required
+          >
+        </label>
+
+        <label for="cars" class="input-label">
+          What are your favorite brand of cars?
+          <select 
+            id="cars" 
+            name="car-srlect" 
+            required
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label for="range" class="input-label">
+          How do you rate our work?
+          <input 
+            id="range" 
+            type="range" 
+            name="range"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="add fieldset">
+        <legend>Additional info:</legend>
+
+        <label for="textarea" class="input-label">
+          Comments:
+          <textarea 
+            id="textarea" 
+            name="textarea"
+          >
+          </textarea>
```

Comment:
now `textarea` has space. 
```suggestion
          <textarea 
            id="textarea" 
            name="textarea"
          ></textarea>
```

Reply:
<img width="286" alt="image" src="https://user-images.githubusercontent.com/102411130/216552735-c9bf04da-c46f-4088-afe1-8dacefb129c2.png">